cmd/kubeconfigwriter/main.go

```diff
@@ -81,8 +81,9 @@ func createKubeconfigFile(resource *v1alpha1.ClusterResource, logger *zap.Sugare
 		Password: pass,
 	}
 	context := &clientcmdapi.Context{
-		Cluster:  resource.Name,
-		AuthInfo: resource.Username,
+		Cluster:   resource.Name,
+		AuthInfo:  resource.Username,
+		Namespace: resource.Namespace,
```

Comment:
What happens if this is an empty string?  Does the field end up in the kubeconfig context with an empty string or is it omitted from the file completely?

Reply:
It's omitted from the file completely if this is empty.
I'll add a comment to code.